gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -163,6 +165,14 @@ def main(args):
         ht = vep_or_lookup_vep(ht, vep_version=args.vep_version)
         ht.write(res.vep_ht.path, overwrite=args.overwrite)
 
+    if args.valid_vep:
+        vep_ht = hl.read_table(
+            resources.run_vep.vep_ht.path
+        )  # TODO: can I use the vep_ht from the previous step?
```

Comment:
yes you can, but you should also add this to the resources function `get_variant_qc_annotation_resources`:

similar to this:
```
    validate_vep = PipelineStepResourceCollection(
        "--validate-vep",
        output_resources={"vep_count_ht": some resource for the output of count_vep_annotated_variants_per_interval},
        pipeline_input_steps=[run_vep],
    )
```

and then change this to use it:
```suggestion
        res = resources.validate_vep
        res.check_resource_existence()
        vep_ht = res.vep_ht.ht()
```